src/server/index.tsx

```diff
@@ -1,14 +1,30 @@
-// @ts-ignore
-const fs = require('fs');
-const path = require('path');
-
-const express = require('express');
-const proxy = require('http-proxy-middleware');
-const cookiesMiddleware = require('universal-cookie-express');
+import fs from 'fs';
+import http from 'http';
+import path from 'path';
+
+import express from 'express';
+import proxy from 'http-proxy-middleware';
+import cookiesMiddleware, {
+  RequestWithCookies,
+} from 'universal-cookie-express';
+
+export type ServerEnvVars = {
+  NODE_ENV: string;
+  PORT: number;
+  REACT_APP_API_HOST: string;
+  REACT_APP_AUTHENTICATION_COOKIE: string;
+  REACT_APP_AUTH_TOKEN_PLACEHOLDER: string;
+  REACT_APP_CRA_PORT: number;
+  REACT_APP_USE_INSECURE_PROXY: boolean;
```

Comment:
Are you sure that this is a `boolean` and not a `string`? I can't find any docs to suggest that this is the case.